README.md

```diff
@@ -74,6 +74,8 @@ manually if your distribution does not provide a package.
 
 First, make sure you have the required __dependencies__:
 - build:
+	- autopoint (for bootstrapping)
+	- autoreconf (for bootstrapping)
```

Comment:
Well I think nowadays there's no need to carry around a massive autogen script, autoreconf do the job as well. That's how it does in GLib and GTK+, and every distro provide these libraries, right ? So I think it's safe to follow their example.

Plus, there are our main dependencies, so if distros can compile GLib and GTK+, then they can